cypress/e2e/filter/reasons.roles.cy.ts

```diff
@@ -0,0 +1,94 @@
+import hashedPassword from "../../fixtures/hashedPassword"
+import { Groups } from "types/GroupName"
+
+function newUserLoginWithGroups(groups: Groups[]) {
+  const user = groups.map((g) => g.toLowerCase()).join("") || "nogroups"
+  const email = `${user}@example.com`
+  cy.task("insertUsers", {
+    users: [
+      {
+        username: user,
+        visibleForces: ["01"],
+        forenames: "Bichard Test User",
+        surname: "01",
+        email: email,
+        password: hashedPassword
+      }
+    ],
+    userGroups: [Groups.NewUI, ...groups]
+  })
+  cy.login(email, "password")
+}
+
+function navigateAndShowFilters() {
```

Comment:
I'm going to be pedantic here:
Would we prefer using arrow functions over "traditional functions"? I think we using arrow functions most of times in this codebase. While in some cases, it just a style choice. For instance I like arrow functions because they have a more concise syntax and quite readable. But there are technical differences too:

- const prevents reassignment but function not
- arrow functions has to be declared before calling it, while functions can be defined after calling them